src/applications/personalization/profile/components/account-security/TwoFactorAuthorizationStatus.jsx

```diff
@@ -20,7 +20,7 @@ const TwoFactorAuthorizationStatus = ({
 
   const mfaHandler = isAuthenticatedWithSSO => {
     recordEvent({ event: 'multifactor-link-clicked' });
-    mfa(isAuthenticatedWithSSO ? 'v1' : 'v0');
+    mfa(isAuthenticatedWithSSO && 'v1');
```

Comment:
can't we just do `mfa()`?

Reply:
> Seems like a lot more cleanup could be done as part of this PR. It seems confusing to leave in vars and props and logical operators when they don't seem to do anything anymore.

So yes, absolutely this can be refactored a lot further.  The reason I was hesitant is that our team (Identity) will eventually create a `v2` sessions controller and I didn't want to have to re-add the version in there. (The v2 session controller timeline is not fleshed out yet).

The other issue was that by doing a larger refactor (which is what I'd like to do) is that a lot of e2e tests would need to be rewritten to account for the SingleSignOnModal appearing after a "successful sign in".  I talked with the Testing Tools team about this and they wanted to wait as they wanted to do a comprehensive change to E2E tests. [Original Slack thread here](https://dsva.slack.com/archives/C01CHAY3ULR/p1631719557146700)

I'm all for removing as much dead code as we can including feature flags we don't need.

Reply:
Okay, thanks for the info.